src/ApiPort/ApiPort.VisualStudio/ApiPortVSPackage.cs

```diff
@@ -98,6 +103,8 @@ protected override void Initialize()
                 OleMenuCommand solutionContextMenuOptionsItem = new OleMenuCommand(ShowOptionsPage, solutionContextMenuOptionsCmdId);
                 solutionContextMenuOptionsItem.BeforeQueryStatus += menuInitializer.SolutionContextMenuItemBeforeQueryStatus;
                 mcs.AddCommand(solutionContextMenuOptionsItem);
+
+                return;
```

Comment:
is this needed?

Reply:
No, it's a ghost of a Task.FromResult that was removed.  I'll remove it.